Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Node.cpp

```diff
@@ -78,6 +81,15 @@ namespace ScriptCanvas
 
     bool NodeVersionConverter(AZ::SerializeContext& context, AZ::SerializeContext::DataElementNode& nodeElementNode)
     {
+        // AddName was mistakenly set to version 1 instead of a higher version than the previous version.
+        // This caused issues with version conversion and nodes, and caused asset processing errors.
+        // This early out check skips conversion that may have previously failed because the version number was mistakenly
+        // set backward and was previously triggering incorrect conversion logic.
+        // This is +1 because the current value is always the latest +1, and when AddName was the most recent, the saved node versions were AddName+1 and not AddName.
+        if (nodeElementNode.GetVersion() == NodeCpp::Version::AddName + 1)
+        {
+            return true;
```

Comment:
Yeah, this is because there are a lot of version update checks below that will fail. They assume that a node coming in at version AddName+1 (2) is the old version 2 and not this new version 2, so they will try to uplift data from the old version 2, and fail because the data doesn't look correct.